signature/src/mechanism/cosign/README.md

```diff
@@ -0,0 +1,207 @@
+# Cosign
+
+Cosign is short for Container Signing, Verification and Storage in an OCI registry.
+It aims to make signatures invisible infrastructure.
+[Cosign](https://github.com/sigstore/cosign/blob/main/README.md).
+
+## Policy Format
+
+Cosign is verified due to the container's policy configuration file.
+
+A Policy Requirement of Cosign should be like this
+
+```json
+{
+    "type": "sigstoreSigned",
+    "keyData": "<PUBKEY-DATA-IN-BASE64>",
+    "keyPath": "<URL-TO-THE-PUBKEY>",
+    "signedIdentity": <JSON-OBJECT>,
+},
+```
+
+Here, 
+* The `type` field must be `sigstoreSigned`, showing that this image is signed by `cosign`.
+* The `keyData` field includes the pubkey's content in base64.
+* The `keyPath` field indicates the pubkey's URL.
+* `signedIdentity` includes a JSON object, refer to [signedIdentity](https://github.com/containers/image/blob/main/docs/containers-policy.json.5.md#sigstoreSigned) for details.
+Because of the mechanism of Cosign, only `matchRepository` and `exactRepository`
+can be used to accept an image.
+
+**WARNING**: Must specify either `keyData` or `keyPath`, and must not both.
```

Comment:
You might want to use a block comment to make the warning stand out, as we do for [Kata](https://github.com/kata-containers/kata-containers/blob/main/docs/Documentation-Requirements.md#warnings-and-other-admonitions):

```suggestion
> **Warning**: Must specify either `keyData` or `keyPath`, but not both.
```

Which renders as:

> **Warning**: Must specify either `keyData` or `keyPath`, but not both.

Oh nice! GH now sees this as a warning and adds a warning symbol too! :smile:

Reply:
Ha XD, brilliant!